src/pages/register.tsx

```diff
@@ -0,0 +1,211 @@
+import Link from 'next/link';
+import Image from 'next/image';
+
+const Register = () => {
+    const inputStyle =
+        'w-full bg-input-gray border-input-gray rounded-full p-2 px-3 mb-4';
+    return (
+        <div className="flex lg:justify-end">
+            <form className=" bg-white w-full lg:w-2/3 mt-40 lg:mt-0 rounded-t-[20px] lg:rounded-none lg:rounded-l-[20px] flex flex-col justify-start items-center text-purple font-ibm">
```

Comment:
- Please avoid using arbitrary numbers like `[20px]` as much as possible. I recommend two options:
    1. Use a tailwind class name which are as close as possible to it such as **3xl**, which represents **24px** and can be used instead.
    2. Extend the `borderRadius` theme in `tailwind.config.js`.
- I believe that for design, large screens should only have rounded corners on the top left. Therefore, please use `rounded-tl-...`.